platform/app/pluginConfig.json

```diff
@@ -88,6 +88,12 @@
   "public": [
     {
       "directory": "./platform/public"
+    },
+    {
```

Comment:
This example shows how to declare a package and to copy a node modules result to the output.